style.css

```diff
@@ -1 +1,157 @@
 /*  Add your styling here */
+
+body{
+    background-color:#f3f3f3;
+}
+*{
+    text-decoration: none;
+}
+header{
+    background-color:#fff;
+    width:100%;
+    height:100px;
+}
+.header-brand{ 
+    font-family:catamaran;
+    font-size:24px;
+    font-weight:1000;
+    color:#111;
+    display:block;
+    margin:auto;
+    text-align:center;
+    padding:20px 0;
+
+}
+header nav ul {
+    display:block;
+    margin:0 auto;
+    width:fit-content;
+}
+header nav ul li {
+    display:inline-block;
+    float:left;
+    list style:none;
+    padding: 0 16px;
+}
+
+header nav ul li a {
+    font-family:catamaran;#
+    font-size:16px;
+    color:#111;
+
+}
+header .header-cases{
+display:none;
+}
+@media only screen and (min-width:768px){
+    .header-brand{ 
+        margin: 0;
+        text-align:left;
+        padding:0px 20px 0px 30px;
+        border-right:3px solid #111;
+        float:left;
+    }
+    header nav ul {
+        margin:20px 0px 0px 20px;
+        float:right;
+    }
+ 
+    header nav ul li a {
+        font-family:catamaran;#
+        font-size:16px;
+        color:#111;
+        line-height:60px
+    }
+    header .header-cases{
+      
+        float:right;
+        }
+}
+.index-banner{
+    width:100%;
+    height: calc(100vh-100px);
+    background-image:url(https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimg1.cookinglight.timeinc.net%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2F1542062283%2Fchocolate-and-cream-layer-cake-1812-cover.jpg%3Fitok%3DrEWL7AIN);
+    background-repeat:no-repeat;
+    background-position:center;
+    background-size:cover
+    display:center;
+}
+.vertical-center{
+display:table-cell;
+vertical-align:middle;
+}
+
+.index-banner h2{
+    font-family:Rboto;
+    font-size:60px;
+    color:#fff;
+    text-align:center;
+    font-weight:900;
+}
+
+.index-banner h1{
+    font-family:Courir;
+    font-size:20px;
+    font-style:italic;
+    line-height:30px;
+    color:#fff;
+    text-align:center;
+    font-weight:100;
+}
+.index-links div{
+    margin:10px 10px 0;
+    width:calc(100%-20px);
+    height:100px;
+    background-color:sandybrown;
+}
+.index-links div h3{
+    font-family:Courir;
+    font-size:20px;
+    font-style:italic;
+    line-height:30px;
+    color:#111;
+    text-align:center;
+    font-weight:100;
+    line-height:100px;
+}
+@media only screen and (min-width:768px){
+    .index-banner{
+        width:100%;
+        height: 400px;
+        background-image:url(https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimg1.cookinglight.timeinc.net%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2F1542062283%2Fchocolate-and-cream-layer-cake-1812-cover.jpg%3Fitok%3DrEWL7AIN);
+        background-repeat:no-repeat;
+        background-position:center;
+        background-size:cover
+        display:center;
+    }
+    .index-banner h1{
+        display:center;
+        width:1000px;
```

Comment:
This is breaking your page layout because the rule applies even when the screen size is smaller than 1000px. 
Can you remove it and make the layout work using flexbox?